File: Clemo97/reverse-integer.py
def reverse_integer(num):
    # Convert the number to a string and handle the sign
    sign = -1 if num < 0 else 1
    num_str = str(abs(num))
    
    # Reverse the string and convert it back to an integer
    reversed_num = int(num_str[::-1])
    
    # Apply the sign and return the result
    return sign * reversed_num

# Test the function
test_cases = [500, -56, -90, 91]
for case in test_cases:
    result = reverse_integer(case)
    print(f"Input: {case}, Output: {result}")

File: MuthoniMN/reverse_integer.js
/**
 * P - int: Number
 * R - reverse version of the integer
 * E - 56 -> 65, -98 -> -89, 
 * **/

function reverseInteger(integer){
  const str = integer >= 0 
              ? `${integer}`.split("").reverse().join("")
              : `-${`${Math.abs(integer)}`.split("").reverse().join("")}`;

  return Number(str);
}

console.log(reverseInteger(500));
console.log(reverseInteger(-56));
console.log(reverseInteger(-90));
console.log(reverseInteger(91));
